indico/modules/rb_new/client/js/common/timeline/TimelineItem.jsx

```diff
@@ -148,7 +148,7 @@ class TimelineItem extends React.Component {
         const {startHour, endHour, onClickCandidate, onClickReservation, room, dayBased} = this.props;
         if (type === 'blocking') {
             segmentStartDt = moment(startHour, 'HH:mm');
-            segmentEndDt = moment(endHour, 'HH:mm');
+            segmentEndDt = (endHour === 24 ? moment('23:59', 'HH:mm') : moment(endHour, 'HH:mm'));
```

Comment:
This is a pretty weird fix and anyway it is not needed as far as I can tell. The issue described in the task

> Element's border is cut out on the right side of the timeline

is solved by two places that you have already fixed in lines 129 and 239.

Reply:
The issue with element border is solved in line 239.
This is a different problem I discovered on the way: blockings don't render properly if timeline `maxHour` is set to 24, because `segmentEndDt.hours()` will resolve to 0, so `segmentWidth` won't be correct

Reply:
Line 129 fixes rendering occurrences that start at timeline maxHour, because we should treat them as out of bounds ones and we didn't

Reply:
Ahh, I see now.